contracts/network/BancorNetwork.sol

```diff
@@ -149,15 +146,18 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
     // a mapping between pools and their respective pool collections
     mapping(Token => IPoolCollection) private _collectionByPool;
 
-    // the pending network fee amount to be burned by the vortex
+    // the pending network fee amount to be burned
     uint256 internal _pendingNetworkFeeAmount;
 
     bool private _depositingEnabled = true;
 
     uint32 private _polRewardsPPM;
 
+    // min network fee amount that can be burned
+    uint256 private _minNetworkFeeBurn;
```

Comment:
I wonder if we really need this. I.e., the caller can opt-in to check whether the accrued fee justifies the gas costs, even if it's only 1 wei. In any case, in the current implementation, the caller will still spend gas for naught, while the `burnNetworkFees` function gracefully returns

Reply:
This was part of the product spec. I'll let @yudilevi answer.

Reply:
Discussed with Yudi, it was actually a DAO decision.

Reply:
Roger